iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -929,7 +928,6 @@ private void mergeDanglingLines(List<MergedLine> lines, DanglingLine dl1, Dangli
 
             // Dangling line 2 must always be reoriented
             // setG1, setB1 and setG2, setB2 will be associated to the end1 and end2 of the reoriented branch
-            ReorientedBranchCharacteristics brp2 = new ReorientedBranchCharacteristics(dl2.getR(), dl2.getX(), dl2.getG(), dl2.getB(), 0.0, 0.0);
```

Comment:
Update the comment. It does not make sense to reorient the branch as in the `danglingLine` only the total shunt admittance (G and B) can be recorded

Reply:
I have removed the comment.